curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/introducing-yourself.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 655283c039eb38f51e0e6f7e
+title: Introducing Yourself
+challengeType: 19
+dashedName: introducing-yourself
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+Tom is introducing himself using a full name and an expression showing he is pleased.
```

Comment:
Just a question. This doesn't seem to be reflected in the correct answer. Should we change it to:

```suggestion
Tom is introducing himself using a full name.
```